faithcomesbyhearing/dbp
# FCBH-1047 Some Gospel Films don't play

# Description
- Added missed environment variables to the `.env.example` file
   - FCBH_AWS_VIDEO_BUCKET
   - FCBH_AWS_BUCKET_LOG

## Issue Link
Original Story: [FCBH-1047](https://fullstacklabs.atlassian.net/browse/FCBH-1047) 

## How Do I QA This
- Add to the `.env` file the `FCBH_AWS_VIDEO_BUCKET=dbp-vid` and verify that the Gospel Films load correctly

## Checklist
